serialization.rst

```diff
@@ -201,9 +205,9 @@ possible mapping values are:
   if it exists, are sent as an associative array to the ``bsonUnserialize``
   function to initialise the object's properties.
 
-  If the class does not implement the ``MongoDB\BSON\Unserializable``
-  interface, then an ``MongoDB\Driver\Exception\UnexpectedValueException``
-  exception is thrown.
+  If the named class does not exist or does not implement the
+  ``MongoDB\BSON\Unserializable`` interface, then an
+  ``MongoDB\Driver\Exception\UnexpectedValueException`` exception is thrown.
```

Comment:
So here you are throwing an exception if the named class does not exist. That makes the case even stronger than doing the same if the class as specified by ``__pclass`` doesn't exist as well.